docker/docker-dev-custom-cfg-kafka/metrictank.ini

```diff
@@ -0,0 +1,288 @@
+# hostpool-simple           : basic pool that tracks which hosts are up and which are not.
+# hostpool-epsilon-greedy   : prefer best hosts, but regularly try other hosts to stay on top of all hosts.
+# tokenaware,roundrobin              : prefer host that has the needed data, fallback to roundrobin.
+# tokenaware,hostpool-simple         : prefer host that has the needed data, fallback to hostpool-simple.
+# tokenaware,hostpool-epsilon-greedy : prefer host that has the needed data, fallback to hostpool-epsilon-greedy.
+cassandra-host-selection-policy = tokenaware,hostpool-epsilon-greedy
+# cassandra timeout in milliseconds
+cassandra-timeout = 1000
+# max number of concurrent reads to cassandra
+cassandra-read-concurrency = 20
+# max number of concurrent writes to cassandra
+cassandra-write-concurrency = 10
+# max number of outstanding reads before blocking. value doesn't matter much
+cassandra-read-queue-size = 100
+# write queue size per cassandra worker. should be large engough to hold all at least the total number of series expected, divided by how many workers you have
+cassandra-write-queue-size = 100000
+# how many times to retry a query before failing it
+cassandra-retries = 0
+# CQL protocol version. cassandra 3.x needs v3 or 4.
+cql-protocol-version = 4
+
+# enable SSL connection to cassandra
+cassandra-ssl = false
+# cassandra CA certficate path when using SSL
+cassandra-ca-path = /etc/raintank/ca.pem
+# host (hostname and server cert) verification when using SSL
+cassandra-host-verification = true
+# enable cassandra user authentication
+cassandra-auth = false
+# username for authentication
+cassandra-username = cassandra
+# password for authentication
+cassandra-password = cassandra
+
+## Profiling and logging ##
+
+# see https://golang.org/pkg/runtime/#SetBlockProfileRate
+block-profile-rate = 0
+# 0 to disable. 1 for max precision (expensive!) see https://golang.org/pkg/runtime/#pkg-variables")
+mem-profile-rate = 524288 # 512*1024
+
+# inspect status frequency. set to 0 to disable
+proftrigger-freq = 60s
+# path to store triggered profiles
+proftrigger-path = /tmp
+# minimum time between triggered profiles
+proftrigger-min-diff = 1h
+# if process consumes this many bytes (see bytes_sys in dashboard), trigger a heap profile for developer diagnosis
+# set it higher than your typical memory usage, but lower than how much RAM the process can take before its get killed
+proftrigger-heap-thresh = 25000000000
+
+# only log log-level and higher. 0=TRACE|1=DEBUG|2=INFO|3=WARN|4=ERROR|5=CRITICAL|6=FATAL
+log-level = 2
+
+# instrumentation stats
+[stats]
+# enable sending graphite messages for instrumentation
+enabled = true
+# stats prefix (will add trailing dot automatically if needed)
+# The default matches what the Grafana dashboard expects
+# $instance will be replaced with the `instance` setting.
+# note, the 3rd word describes the environment you deployed in.
+prefix = metrictank.stats.defaulte.$instance
```

Comment:
this was because i wanted to make sure the environment and instance showed up in the right place (`default.default` was a bit ambiguous). but I can undo that now and set both to default